libcnb-data/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "libcnb-data"
+version = "0.3.0"
```

Comment:
Should this start at `0.1.0` given it's a new crate and we presumably won't keep them in lock-step forever? I can't decide / happy either way.

Reply:
+1 on being unsure. It's certainly easier for now to keep the versions in lockstep to reduce possible confusion.